src/components/vue-form-builder.vue

```diff
@@ -469,15 +469,10 @@ $header-bg: #f7f7f7;
   .mask {
```

Comment:
Could we use the bootstrap helper classes for width, height, and position?

Reply:
I thought about this; we could use `w-100 h-100 position-absolute`, but I didn't see any helpers for `top` and `left` positioning. I wanted to keep the `mask` class since it gives the div meaning, and makes it easier to find when debugging.

I wonder if we can use something like `pointer-events: none` instead of a mask though; will look into it and update PR.

Reply:
I like `pointer-events: none` instead of defining what percent of the mask should cover.